src/components/Table/common/utils.ts

```diff
@@ -243,6 +222,7 @@ export function getGridTemplateColumns<T extends RowData>(
   columns: Column<T>[]
 ): string {
   return columns
+    .filter(filterGroupedColumn)
```

Comment:
We filter out any grouped column from the `gridTemplateColumns` value - the table renders a grouped column as a as a row.